doc/nrf/ug_matter_overview_device_types.rst

```diff
@@ -0,0 +1,210 @@
++===================+==============================================================================+=====================+=======================================+
+| Door Lock         | A Door Lock is a device used to secure a door. It is possible to actuate     | Certifiable         | :ref:`matter_lock_sample`             |
+|                   | a door lock either by means of a manual or a remote method.                  |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Door Lock         | A Door Lock Controller is a device capable of controlling a door lock.       | Certifiable         |                                       |
+| Controller        |                                                                              |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Window Covering   | A Window Covering is a device used to control absolute position of window    | Certifiable         | :ref:`matter_window_covering_sample`  |
+|                   | cover.                                                                       |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Window Covering   | A Window Covering Controller is a device that controls an automatic window   | Certifiable         |                                       |
+| Controller        | covering.                                                                    |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+
+.. _ug_matter_device_types_hvac:
+
+HVAC device types
+=================
+
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Device type       | Description (from Device Library specification)                              | State of the device | Dedicated sample in the |NCS|         |
++===================+==============================================================================+=====================+=======================================+
+| Heating/Cooling   | A Heating/Cooling Unit is a device capable of heating or cooling a space     | Certifiable         |                                       |
```

Comment:
@doublemis1 could you confirm that? I think I heard about adding support to this device type in Matter 1.1, so I'm not sure if that was certified. The same applies to Fan controller.

Reply:
You are right:
List of Provisional Items
The following is a list of provisional items.
• Support for Heating/Cooling Unit is provisional.
• Support for Fan device type is provisional.

Need to verify if the pressure sensor is set to provisional

Reply:
Changed both to Provisional.